app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -12,7 +12,8 @@ import org.oppia.app.settings.profile.ProfileListActivity
 import javax.inject.Inject
 
 const val SELECTED_CONTROLS_TITLE_KEY = "SELECTED_CONTROLS_TITLE_KEY"
-const val LAST_LOADED_FRAGMENT_KEY = "LAST_LOADED_FRAGMENT_KEY"
+const val LAST_LOADED_FRAGMENT_SAVED_KEY =
```

Comment:
This string is correct but it should be mentioned in the companion object.

```
  companion object {
    internal const val LAST_LOADED_FRAGMENT_SAVED_KEY =
      "AdministratorControlsActivity.last_loaded_fragment"
    fun createAdministratorControlsActivityIntent(context: Context, profileId: Int?): Intent {
      val intent = Intent(context, AdministratorControlsActivity::class.java)
      intent.putExtra(KEY_NAVIGATION_PROFILE_ID, profileId)
      return intent
    }

    fun getIntentKey(): String {
      return KEY_NAVIGATION_PROFILE_ID
    }
  }
```


Once you have mentioned it in companion object you can use it anywhere like this:
`AdministratorControlsActivity.LAST_LOADED_FRAGMENT_SAVED_KEY`

Reply:
done